packages/react/src/components/form/form.test.tsx

```diff
@@ -57,9 +57,11 @@ describe("#Form", () => {
     );
     user.click(cancelButton);
 
-    await expect(
-      screen.findByTestId("sid-form-initial-state")
-    ).resolves.toBeInTheDocument();
+    await waitFor(() =>
```

Comment:
This won't solve the problem - `findByTestId` internally already uses `waitFor`. Problem is with the test setup that is related to timers. We can solve that in a separate PR in order to keep this one clean.